src/components/sidebarMenu/sidebarMenu.tsx

```diff
@@ -6,9 +6,14 @@ import useSidebarStore from '@/store/useSidebarStore';
 const SidebarMenu = () => {
   const { isSidebarOpen, toggleSidebar } = useSidebarStore();
 
+  const handleSidebar = (): void => {
+    toggleSidebar();
+    !isSidebarOpen ? (document.body.style.overflow = 'hidden') : (document.body.style.overflow = 'auto');
+  };
+
```

Comment:
if a user opens the sidebar on a mobile or whatever just open sidebar and navigates to the main page, they will find that scrolling on the main page itself is disabled. Additionally preventing scrolling within the sidebar itself is a critical drawback.